frontend/src/theme/components/Checkbox.ts

```diff
@@ -0,0 +1,53 @@
+import { ComponentStyleConfig } from '@chakra-ui/react'
+import { getColor } from '@chakra-ui/theme-tools'
+
+export const Checkbox: ComponentStyleConfig = {
+  parts: ['others'],
+  baseStyle: ({ theme }) => ({
+    control: {
+      borderRadius: '0.19rem',
+      border: '2px solid',
+      borderColor: 'primary.500',
+      _checked: {
+        bg: 'primary.500',
+        borderColor: 'primary.500',
+        _hover: {
+          bg: 'primary.500',
+          borderColor: 'primary.500',
+        },
+      },
+      _focus: {
+        boxShadow: 'none',
+      },
+    },
+    label: {
+      textColor: 'secondary.700',
+      mx: '1rem',
```

Comment:
I think this is supposed to be `ml`? There does not seem to be a right margin on figma